src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/queue/TempConfig.java

```diff
@@ -0,0 +1,46 @@
+package ca.bc.gov.open.jag.efilingreviewerapi.queue;
+
+import ca.bc.gov.open.clamav.starter.ClamAvService;
+import ca.bc.gov.open.efilingdiligenclient.diligen.DiligenService;
+import ca.bc.gov.open.jag.efilingreviewerapi.document.DocumentsApiDelegateImpl;
+import ca.bc.gov.open.jag.efilingreviewerapi.extract.mappers.ExtractRequestMapper;
+import ca.bc.gov.open.jag.efilingreviewerapi.extract.store.ExtractStore;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.connection.jedis.JedisConnectionFactory;
+import org.springframework.data.redis.core.StringRedisTemplate;
+import org.springframework.data.redis.listener.PatternTopic;
+import org.springframework.data.redis.listener.RedisMessageListenerContainer;
+import org.springframework.data.redis.listener.adapter.MessageListenerAdapter;
+
+@Configuration
+@EnableCaching
+public class TempConfig {
+
+    @Bean
+    RedisMessageListenerContainer container(JedisConnectionFactory jedisConnectionFactory,
+                                            MessageListenerAdapter listenerAdapter) {
+
+        RedisMessageListenerContainer container = new RedisMessageListenerContainer();
+        container.setConnectionFactory(jedisConnectionFactory);
+        container.addMessageListener(listenerAdapter, new PatternTopic("documentWait"));
+
+        return container;
+    }
+
+    @Bean
+    MessageListenerAdapter listenerAdapter(Receiver receiver) {
+        return new MessageListenerAdapter(receiver, "receiveMessage");
+    }
+
+    @Bean
+    public StringRedisTemplate stringRedisTemplate(JedisConnectionFactory jedisConnectionFactory) {
+        return new StringRedisTemplate(jedisConnectionFactory);
+    }
+
+    @Bean
+    public Receiver receiver(DiligenService diligenService, ExtractRequestMapper extractRequestMapper, ExtractStore extractStore, StringRedisTemplate stringRedisTemplate, ClamAvService clamAvService) {
+        return new Receiver(120, new DocumentsApiDelegateImpl(diligenService, extractRequestMapper, extractStore, stringRedisTemplate, clamAvService));
```

Comment:
This should be a property.  But Since this code will be deleted soon I wanted minimize the cleanup.

Reply:
Also thinking we could consider reducing the wait time and re-queue the event when throws a not found(Document not processed).

Reply:
agree, it can be hardcode now, of you can use the @Value pattern if you want